README.md

```diff
@@ -1 +1,53 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Portfolio
+> ![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Open your terminal or command line 
+- Clone the project using:
+
+```
+$ git clone https://github.com/praizerema/portfolio.git
+```
+
+### Prerequisites
+
+- A text editor or and IDE
+- Git installed
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] [OPTIONAL] Instead of using default name you should use your name, for instance:

What you have:
```
👤 **Author1**
```
What you can add:
```
👤 **Your Name**
```